contracts/votingTokens/ZeroVotingERC20.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "./IZeroVotingERC20.sol";
+
+/**
+* Throw this error if someone submits a zero token burn address.
+* @param to address to send the transaction.
+*/
+error InvalidBurnAddress(address to);
+
+contract ZeroVotingERC20 is ERC20, ERC20Permit, ERC20Votes, AccessControl, IZeroVotingERC20 {
```

Comment:
we don't need both ERC20 and ERC20Permit here, since ERC20Permit already inherits from ERC20